AdsLib/AdsDef.h

```diff
@@ -245,15 +245,26 @@ struct AmsNetId {
     /** NetId, consisting of 6 digits. */
     uint8_t b[6];
 
-#ifdef __cplusplus

```

Comment:
This is a public header, which should be usable in pure C projects. Without this guard a C compiler will complain.